plugins/inventory/libvirt.py

```diff
@@ -118,7 +144,32 @@ def parse(self, inventory, loader, path, cache=True):
             self.inventory.add_group(inventory_hostname_alias)
             self.inventory.add_child(inventory_hostname_alias, inventory_hostname)
 
-            if connection_plugin is not None:
+            # Set the interface information.
+            ifaces = {}
+
+            for iface, iface_info in (connection.lookupByName(server.name())).interfaceAddresses(
```

Comment:
This is a lot of work to go through to throw away if the connection plugin is going to be used. Are we doing this so that we add the interface information to the inventory, whether we use the connection plugin or not?

Reply:
Yes, it was my intention to have the interface information available in both cases: connection plugin on or off.I can leave it out in case that's a bit overhead. But I can imagine that in some cases you would like to know this kind of information (if someone would like to make modifications within libvirt itself for example).

Another option is to leave it out when the connection plugin is on and add it later on back, together with all the other information that's available from libvirt about a host; so you have a full set of information about each host from libvirt. For example: CPU info, RAM info, DISK info, etc.

I'm fine with any of these options. Or if you have a better one, please let me know!